.gitattributes

```diff
@@ -0,0 +1,2 @@
+# https://prettier.io/docs/en/options.html#end-of-line
+* text=auto eol=lf
```

Comment:
Without this Prettier loses its shit on Windows. 🤷

![Screen Shot 2022-11-08 at 4 32 52 PM](https://user-images.githubusercontent.com/33762/200709723-9f961f66-6f53-4044-8fde-ebd1f450d5be.png)